pkgs/development/zig-packages/build-zig-package/default.nix

```
@@ -0,0 +1,42 @@
+{ lib, stdenv, zig, mktemp }:
+
+{ strictDeps ? true
+, nativeBuildInputs ? [ ]
+, enableParallelBuilding ? true
+, meta ? { }
+
+, zigDefaultBuildFlags ? "-Drelease-safe -Dcpu=baseline"
+, zigBuildFlags ? ""
+, ...
+}@attrs:
+
+stdenv.mkDerivation (attrs // {
+  inherit strictDeps enableParallelBuilding;
+  nativeBuildInputs = [ zig ] ++ nativeBuildInputs;
+
+  postConfigure = attrs.postConfigure or ''
+    export HOME=$(mktemp -d)
+  '';
+
+  buildPhase = attrs.buildPhase or ''
+    runHook preBuild
+    zig build ${zigDefaultBuildFlags} ${zigBuildFlags}
```

Now I do not remember perfectly, but something like

```suggestion
    zig build ${lib.concatStringsSep " " (zigDefaultBuildFlags ++ zigBuildFlags)}
```

---

this is a no-op